### PR TITLE
feat(core/message): add options to set size and position

### LINKED ISSUE #1071: Modal Positioning
### Prerequisites

- [X] I have not leaked any internal company information.
- [X] I have not posted internal links or code snippets.

### Suggestion / feature request

It would be nice to have the possibility of positioning the modal by passing in a value into the service.

For example passing in position="top left", which would result in the Modal being placed at the top left of the screen.

### LINKED ISSUE #1211: Configurable size for showMessage
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### Suggestion / feature request

With introduction of the modal showMessage there was saved a lot of time and code for simple message modal dialogs.
Due to our UX concept, the pre-configured size is not our first choice. We would appreciate the possibility to have a parameter to configure the dialog size.

### PR DESCRIPTION
**Pull request by nuke-ellington**
## 💡 What is the current behavior?

GitHub Issue Number: Closes #1211, Closes #1071, IX-1149

## 🆕 What is the new behavior?

-
-

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [ ] 🦮 Accessibility (a11y) features were implemented
- [ ] 🗺️ Internationalization (i18n) - no hard coded strings
- [ ] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [ ] 📄 Documentation was reviewed/updated (`pnpm run docs`)
- [x] 🧪 Unit tests were added/updated and pass (`pnpm test`)
- [x] 📸 Visual regression tests were added/updated and pass ([Guide](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing))
- [x] 🧐 Static code analysis passes (`pnpm lint`)
- [x] 🏗️ Successful compilation (`pnpm build`, changes pushed)

### CODE REVIEW

File: packages/core/src/components/utils/modal/message.ts [hunk 1]
```diff
@@ -55,6 +57,10 @@ function createConfirmButtons(
   ];
 }
 
+export type MessageConfig<T> =
+  | Omit<ModalConfig<T, unknown>, 'content' | 'title'>
+  | MessageContent;
+
```
Comment (danielleroux) [inline packages/core/src/components/utils/modal/message.ts:63]: Type is never used anywhere

File: packages/core/src/components/utils/modal/message.ts [hunk 2]
```diff
@@ -32,7 +34,7 @@ function createConfirmButtons(
   payloadOkay?: any,
   payloadCancel?: any
 ) {
-  let actions = [];
+  let actions: any[] = [];
```
Comment (danielleroux) [inline packages/core/src/components/utils/modal/message.ts:37]: no any!

**Review by danielleroux — CHANGES_REQUESTED** [review]

File: packages/core/src/components/utils/modal/message.ts [hunk 3]
```diff
@@ -53,25 +53,23 @@ function createConfirmButtons(
...
       type: 'okay',
       payload: payloadOkay,
-    },
+    } as MessageAction,
   ];
 }
 
-export type MessageConfig<T> =
-  | Omit<ModalConfig<T, unknown>, 'content' | 'title'>
-  | MessageContent;
```
Comment (danielleroux) [inline packages/core/src/components/utils/modal/message.ts]: Why did you removed this type? Its a good idea in general to have a MessageConfig which reflects the ModalConfig

File: packages/core/src/components/utils/modal/message.ts [hunk 4]
```diff
@@ -51,21 +53,25 @@ function createConfirmButtons(
...
   message: string;
-  actions: {
-    id: string;
-    type: 'button-primary' | 'button-secondary' | 'okay' | 'cancel';
-    text: string;
-    payload?: any;
-  }[];
+  actions: MessageAction[];
+  centered?: boolean;
+  size?: IxModalSize;
```
Comment (danielleroux) [inline packages/core/src/components/utils/modal/message.ts:74]: We should export type in the utils/modal/index file

**Review by danielleroux — CHANGES_REQUESTED** [review]

File: packages/core/src/tests/message/mesage.e2e.ts
Comment (AndreasBerliner) [inline packages/core/src/tests/message/mesage.e2e.ts:1]: filename has a typo

**Review by danielleroux — APPROVED** [review]

**Pull request merged**